mod-pubsub-server/src/test/java/org/folio/kafka/PubSubConfigTest.java

```diff
@@ -0,0 +1,43 @@
+package org.folio.kafka;
+
+import org.junit.After;
+import org.junit.Test;
+
+import java.util.function.Supplier;
+
+import static org.junit.Assert.*;
```

Comment:
please import only the necessary classes instead of the entire package

Reply:
Done